vsts/scripts/publishFilesToAzureStorage.sh

```diff
@@ -70,6 +72,18 @@ uploadFiles() {
     done
 }
 
+storageAccountUrl="https://$storageAccount.blob.core.windows.net"
```

Comment:
nit: similar comment as before but specific to this file, would you mind renaming `storageAccount` to `storageAccountName`?